### DISCUSSION TITLE
ZSH: command not found: gyb

### DISCUSSION THREAD

**Discussion by geoffcheckfront** [GAM-team/got-your-back#382] [Q&A]
Installed on Macbook Pro M1, GAM working. Installed gyb successfully but when running command it is not recognized. Did a update install as well, successful, but still not luck. Wondering how I can get gyb to work.

**Comment by alexbreen7** [comment]
I ran into this problem also. I was able to solve it by referencing the executable directly. eg instead of running 
`gyb --email youremail@gmail.com` 
as instructed in the documentation, I ran:

`/Users/user/bin/gyb/gyb  --email youremail@gmail.com`

(with my username in place of "user". And this seemed to do the trick)
